### DISCUSSION TITLE
TeslaLogger work fine on OrangePi 3 LTS

### DISCUSSION THREAD

**Discussion by Tobor7** [bassmaster187/TeslaLogger#927] [General]
Since the Raspberry Pi are hard to find for over a year, I managed to setup TeslaLogger on a Orange Pi. Here is my setup:
- Orange Pi 3 LTS (will probably work on a Orange Pi Zero 2, but did not try)
- Armbian Bullseye (I prefer this distribution over the OrangePi Debian/Ubuntu, for the continuous update of Armbian. Note that Armbian Jammy v22.11 have a bug with the wifi driver and must use an ethernet cable. The wifi is ok with Bullseye kernel 5.15y)
- Docker version  20.10.23
- TeslaLogger version 1.54.11
- htop info:
![image](https://user-images.githubusercontent.com/90622675/216499470-f61290f8-6240-470b-bdf0-c9f537909b4f.png)

**Comment by tommymann1982** [comment]
I am using Orangepi 4 LTS, but had to use  the Armbian 24.5.1 Bookworm. 

There are some issues though: 
- after last update (1 month ago) no desktop availaible (in the old Armbian version it did work)
- The OrangePi  destroyed one sd card (Ive lost all backups too)
- The OrganePi has a slightli higher consumption than a raspberry pi
- I would like to use Raspberry OS instead of Armbian, but this is not possibel.

**Comment by Tobor7** [comment]
I found that using an orange pi 3 LTS is overkill to only run Teslalogger, it run fine with an orange pi zero 2 or 3 with at least 1 mo of ram. For over 8 month I use an orange pi 2 with Armbian 23.02.2 Bulleye (command line only) without problems. The pi zero 2 is directly powered by the USB port of my router. I also made a configuration for my friend on an orange pi zero 3 an it run flawlessly. At 20$ for an orange pi 3 on Aliexpress, it is hard to find a cheaper solution!
